mcstatus/pinger.py

```diff
@@ -35,7 +35,7 @@ class RawResponseDescriptionWhenDict(TypedDict, total=False):
         underlined: bool
         obfuscated: bool
 
-    RawResponseDescription: TypeAlias = Union[RawResponseDescriptionWhenDict, list[RawResponseDescriptionWhenDict], str]
+    RawResponseDescription: TypeAlias = RawResponseDescriptionWhenDict | list[RawResponseDescriptionWhenDict] | str  # type: ignore # noqa: E501
```

Comment:
Similarly to my previous remark, this typealias should be a string, let's not use type ignores here.